### ISSUE TITLE
RGBD to 3D PointCloud

### ISSUE THREAD

**Issue by lynetcha** [ScanNet/ScanNet#1]
Hi,

Do you have code to generate annotated 3D Point Clouds of the areas?

Thanks,

Lyne

**Comment by Coldplayplay** [comment]
yeah,  I am looking for methods of making semantic segmentation label directly on own point cloud data too, anyone has suggestions?
